docs/aries-cloud-agent-demo.md

```diff
@@ -0,0 +1,241 @@
+```
+
+Again, make sure the above ports are unused and take note of the specific IP addresses and ports. In this case:
+
+* Alice admin IP: 127.0.0.1
+* Alice admin port: 8031
+* Alice webhook port: 8032
+
+## Create Alice and Faber AEAs
+
+### Create Alice_AEA
+
+In the third terminal, create an Alice_AEA and move into its project folder: 
+
+``` bash
+aea create alice
+cd faber
+```
+
+### Add and Configure the Skill
+
+Add the `aries_alice` skill:
+
+``` bash
+aea add --local skill fetchai/aries_alice:0.1.0
+```
+
+You then need to configure this skill. Open the skill's configuration file in `alice/vendor/fetchai/skills/aries_alice/skill.yaml` and ensure `admin_host` and `admin_port` details match those you noted above for Faber_ACA. 
+
+### Add and Configure the Connections
+
+Add `http_client`, `oef` and `webhook` connections:
+
+``` bash
+aea add --local connection fetchai/http_client:0.1.0
+aea add --local connection fetchai/webhook:0.1.0
+aea add connection fetchai/oef:0.1.0
+```
+
+You now need to configure the `webhook` connection. Open the connection's configuration file in `alice/vendor/fetchai/connections/webhook/connection.yaml` and ensure `webhook_port` matches with what you used above for Faber_ACA. 
+
+Next, make sure that the value of `webhook_url_path` is `/webhooks/topic/{topic}/`.
+
+### Configure Alice_AEA:
+
+You now need to ensure that Alice_AEA uses the OEF connection as its default connection. Open the agent's configuration file in `alice/aea-config.yaml` and ensure that the `default_connection`'s value is `fetchai/oef:0.1.0`.
+
+### Install the Dependencies and Run Alice_AEA:
+
+Install the dependencies:
+
+``` bash
+aea install
+```
+
+Then run Alice_AEA:
+
+``` bash
+aea run --connections fetchai/http_client:0.1.0,fetchai/oef:0.1.0,fetchai/webhook:0.1.0
+```
+
+You should see Alice_AEA running and showing its identity on the terminal. For example:
+
+``` bash
+My address is: YrP7H2qdCb3VyPwpQa53o39cWCDHhVcjwCtJLes6HKWM8FpVK
+```
+
+Make note of this value. We will refer to this as Alice_AEA's address.
+
+### Create Faber_AEA: 
+
+In the fourth terminal, create a Faber_AEA and move into its project folder: 
+
+``` bash
+aea create faber
+cd faber
+```
+
+### Add and Configure the Skill:
+
+Add the `aries_faber` skill:
+
+``` bash
+aea add --local skill fetchai/aries_faber:0.1.0
+```
+
+You then need to configure this skill. Open the skill's configuration file in `faber/vendor/fetchai/skills/aries_alice/skill.yaml` and ensure `admin_host` and `admin_port` details match those you noted above for Faber_ACA. In addition, make sure that the value of `alice_id` matches Alice_AEA's address as seen in the third terminal.
+
+### Add and Configure the Connections:
+
+Add `http_client`, `oef` and `webhook` connections:
+
+``` bash
+aea add --local connection fetchai/http_client:0.1.0
```

Comment:
no local

Reply:
Done. 

(I needed local at the time because of the local change I had to make to http_client)